reporter-opentsdb/src/main/java/io/ultrabrew/metrics/reporters/opentsdb/OpenTSDBHttpClient.java

```diff
@@ -73,10 +75,33 @@ private CloseableHttpClient getHttpClient() {
         .build();
   }
 
+  @SuppressFBWarnings(value="NP", justification="Null check IS performed on tags")
   void write(final String metricName,
       final String[] tags,
       final long timestamp,
       final String value) throws IOException {
+    // validation
+    if (metricName == null || metricName.isEmpty()) {
```

Comment:
This "null or empty" is repeated so many times that it would probably make sense to add a helper method for it.

Reply:
If we have Guava already we can use the `Strings.isNullorEmpty` helper. Or maybe some Apache lib has it. I'll poke around. Don't want to add a new dependency.

Reply:
Let's put this as a utility method.
```
 public static boolean isNullOrEmpty(String s) {
    return s == null || s.isEmpty();
  }
```

Reply:
Absolutely no Guava anywhere near this library

Reply:
@lonemeow Fixed, mind reviewing please?